auth_oauth_multi_token/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2016 Florent de Labarre
+# Copyright 2017 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{
+    'name': 'OAuth Multi Token',
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'Florent de Labarre, '
+              'Camptocamp, '
+              'Odoo Community Association (OCA)',
```

Comment:
```suggestion
              'Odoo Community Association (OCA)'
```

Reply:
That comma is needed for proper dictionary format (it's outside the quote).

Reply:
Sorry, I see that they have added the new author after this...

But I don't consider this kind of migration should rise them to co-authors, so the solution is to remove that author.